packages/core/src/dialects/abstract/query-interface.d.ts

```diff
@@ -295,7 +291,12 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
    *
    * @param options
    */
-  dropAllTables(options?: QueryInterfaceDropAllTablesOptions): Promise<void>;
+  dropAllTables(options?: QiDropAllTablesOptions): Promise<void>;
+
+  /**
+   * Internal Drop All Tables method
+   */
+  _dropAllTables(tableNames: string[], skip: string[], options?: QiDropAllTablesOptions): Promise<void>;
```

Comment:
this only exists in sqlite. It could be a method in `SqliteQueryInterfaceInternal` instead

Reply:
`_dropAllTables` is used but the `dropAllTables` method in the `AbstractQueryInterface` to loop over the tables and drop them, it's not just used by SQLite.

This function could be merged into the `dropAllTables` method as this is the only use of it.

I can add a TODO if you want or make the change in this PR.

Reply:
I've submitted PR #16452 which merges the `dropAllTables` and `_dropAllTables` queries